src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/ServiceController.php

```diff
@@ -260,6 +260,8 @@ public function deleteAction(Request $request, $serviceId)
             $this->authorizationService->getActiveServiceId()
         );
 
+        $entityList = $this->entityService->getEntityListForService($service);
```

Comment:
It should be more efficient to fetch the entities after the form is handled.

Reply:
Good point! Moved it